app/src/pages/Lobby/index.tsx

```diff
@@ -45,55 +49,68 @@ function SettingsSection() {
   const currentPlayer = React.useContext(CurrentPlayerContext)
   const titleClasses = useTitleStyle()
   return (
-    <Grid item>
-      <Grid container spacing={2} direction="column">
-        <Grid item>
-          <Typography variant="h4" className={titleClasses.title}>
-            Settings
-          </Typography>
-          {currentPlayer.role === PlayerRole.Participant && (
-            <div style={{ color: grey[600] }}>
-              (Only your host can set these)
-            </div>
-          )}
-        </Grid>
-        <Grid item>
-          <SecondsPerTurnInput
-            value={String(currentGame.seconds_per_turn || "")}
-          ></SecondsPerTurnInput>
-        </Grid>
-        <Grid item>
```

Comment:
btw I removed this section that says "Card"

Reply:
I think we can maybe consider adding back when we have more Turns and Cards configs, I was looking for some style consistency with the "rounds" part - and introduce the vocabulary more